src/app/ngrx/effects/castle.effects.spec.ts

```diff
@@ -199,27 +165,232 @@ describe('CastleEffects', () => {
     const action = {
       type: ACTIONS.ActionTypes.CASTLE_EPISODE_METRICS_LOAD,
       payload: {
-        seriesId: podcasts[0].seriesId,
-        page: episode.page,
-        id: episode.id,
-        guid: episode.guid,
+        podcastId: episodes[0].podcastId,
+        page: episodes[0].page,
+        guid: episodes[0].guid,
         metricsType: METRICSTYPE_DOWNLOADS,
         interval: INTERVAL_DAILY,
         beginDate: new Date(),
         endDate: new Date()
       }
     };
     const success = new ACTIONS.CastleEpisodeMetricsSuccessAction({
-      seriesId: podcasts[0].seriesId,
-      page: episode.page,
-      id: episode.id,
-      guid: episode.guid,
+      podcastId: episodes[0].podcastId,
+      page: episodes[0].page,
+      guid: episodes[0].guid,
       metricsPropertyName: getMetricsProperty(INTERVAL_DAILY, <MetricsType>METRICSTYPE_DOWNLOADS),
-      metrics: downloads
+      metrics: ep0Downloads
     });
 
     actions$.stream = hot('-a', { a: action });
     const expected = cold('-r', { r: success });
     expect(effects.loadEpisodeMetrics$).toBeObservable(expected);
   });
+
+  it('should startWith loading page 1 of podcasts', () => {
+   const success = new ACTIONS.CastlePodcastPageSuccessAction({
+      podcasts: [podcast],
+      page: 1,
+      total: 1,
+      all: true
+    });
+
+    actions$.stream = hot('-');
```

Comment:
kinda subtle here (my least favorite kind, but it's honestly pretty lean and works well for hydrating the app state), but because the `loadPodcastPage$` effect has the startWith operator, NgRx fires it on application load. so in this marble test, we're not giving it a load action because the effect does that for us automagically on load.